### PR TITLE
Fixed Spelling.

### PR DESCRIPTION
**Pull request by jimmycasey**
**Please** add a meaningful description for your change here

Hi there, 
I wrote this bot based on the top 10 most misspelled english words.
(https://blog.oxforddictionaries.com/2016/08/02/corpus-misspellings/)
The bot queries, forks, clones, fixes and commits but I manually create the PRs.
All the best,
Jimmy

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | --- | --- | --- | ---

### CODE REVIEW

File: website/src/js/language-switch.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ $(document).ready(function() {
             /**
              * @desc Search next sibling and if it's also a code block, then store
                     it's type and move onto the next element. It will keep
-                    looking untill their is no direct code block decendent left.
+                    looking until their is no direct code block decendent left.
```
Comment (Ardagan) [inline website/src/js/language-switch.js]: ...looking until there is...

Comment (jimmycasey) [reply website/src/js/language-switch.js]: Done

**Comment by huygaa11** [comment]
@committer please merge.

**Comment by apilloud** [comment]
The source of truth for the beam website is still http://github.com/apache/beam-site. Looks like there is a PR there already: https://github.com/apache/beam-site/pull/515

**Pull request closed without merging**